source/scale/push-notification-health-targets.rst

```diff
@@ -0,0 +1,86 @@
+Push notification health targets
+=================================
+
+.. include:: ../_static/badges/ent-cloud-selfhosted.rst
+  :start-after: :nosearch:
+
+When using the `Mattermost Notification Health <https://grafana.com/grafana/dashboards/21305-mattermost-notification-health>`_ Grafana dashboard to track different types of notifications sent from Mattermost, we recommend adhering to the following mobile push notification health targets to ensure a performant production deployment of Mattermost.
+
+.. note::
+
+    - Accessing and enabling Mattermost Notification Health Monitoring requires the feature flag ``NotificationMonitoring`` to be set to ``true``. 
+    - System admins can :ref:`disable notification monitoring data collection <configure/site-configuration-settings:enable notification monitoring>` through the System Console.
+
+Push Proxy Delivery Rate
+------------------------
+
+The **Push Proxy Delivery Rate** panel indicates the percentage of push notifications that have been successfully delivered to the endpoints (i.e., mobile devices) out of the total number of push notifications sent.
```

Comment:
Not exactly the devices. It is more about the Push Notification Service (Apple Push Notification service (APNs) or Firebase Cloud Messaging (FCM)).